src/index.html

```diff
@@ -1,12 +1,381 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no,
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
+    <title>Moyo catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="content">
+    <header class="content__header header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="Moyo-logo"
+          class="header__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog-page">
+        <div class="catalog-page__wrapper">
+          <div class="catalog-page__product-card product-card" data-qa="card">
+            <img
+              class="product-card__product-img"
+              src="./images/imac.jpeg"
+              alt="product-picture"
+            >
+            <h2 class="product-card__product-name">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+            <p class="product-card__vendor-code">
+              Код товара: 195434
+            </p>
+            <div class="product-card__rating">
+              <div class="stars stars_4">
```

Comment:
```suggestion
              <div class="stars stars--4">
```
by default we using `--` for modifier if you want to use `_` you need to change it in bemlint.json 
modifier divider prop